src/components/Map.astro

```diff
@@ -1,10 +1,10 @@
-export const Map = () => (
+  <Fragment is:raw>
   <iframe
     src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d10245.064741908278!2d14.4290206!3d50.0625764!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x4d26855708eb61f7!2sPrague%20Congress%20Centre!5e0!3m2!1sen!2suk!4v1676555813425!5m2!1sen!2suk"
     title="Map"
-    className="w-full aspect-video max-w-[800px] border-4 border-white rounded-lg shadow-lg"
-    allowFullScreen={true}
+    class="w-full aspect-video max-w-[800px] border-4 border-white rounded-lg shadow-lg"
+    allowFullScreen="true"
     loading="lazy"
     referrerPolicy="no-referrer-when-downgrade"
   ></iframe>
-);
+  </Fragment>
```

Comment:
FYI I updated this in another PR: https://github.com/EuroPython/website/pull/1227/files#diff-d7d44fd3cc527b8dbcaa8b661a0eb769d94080a4ae4fccc535320ffdc9fa5fefR1

We need to have the component map generic and be able to pass coordinates from google maps, so it's reusable at multiple places